hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSBucketLayout.java

```diff
@@ -116,19 +118,21 @@ public static void initCluster() throws Exception {
         .setNumDatanodes(3)
         .build();
     cluster.waitForClusterToBeReady();
-    objectStore = cluster.getClient().getObjectStore();
+    client = cluster.getClient();
+    objectStore = client.getObjectStore();
     rootPath = String.format("%s://%s/",
         OzoneConsts.OZONE_OFS_URI_SCHEME, conf.get(OZONE_OM_ADDRESS_KEY));
 
     // create a volume and a bucket to be used by RootedOzoneFileSystem (OFS)
     volumeName =
-        TestDataUtil.createVolumeAndBucket(cluster)
+        TestDataUtil.createVolumeAndBucket(client)
             .getVolumeName();
     volumePath = new Path(OZONE_URI_DELIMITER, volumeName);
   }
 
   @AfterClass
   public static void teardown() throws IOException {
+    IOUtils.closeQuietly(client);
```

Comment:
nit: I'm not sure if `closeQuietly` is null safe (I think when a `closeQuietly(null)` will pass a null array instead of an array of a null object). If it is, either move this statement into the `if (cluster != null) {` block or make `closeQuietly` null-safe.

Reply:
Since `client` is declared as `OzoneClient`, it will be a `null` instance of that type.

Added a test (with `ByteArrayOutputStream` as `hdds-common` doesn't know `OzoneClient`) to verify.

Also added `null` check for array.